himan-plugins/source/grib.cpp

```diff
@@ -705,6 +671,10 @@ void WriteAreaAndGrid(NFmiGribMessage& message, const shared_ptr<himan::grid>& g
 				{
 					message.SetLongKey("shapeOfTheEarth", 0);
 				}
+				else if (earth.A() == 6371229)
```

Comment:
Maybe also check that A() == B() so we know it's a sphere.

Reply:
It's actually checked couple of lines above but not part of the diff.